src/test/java/no/unit/nva/cristin/projects/FetchCristinProjectsTest.java

```diff
@@ -84,47 +84,47 @@ void handlerReturnsExpectedBodyWhenRequestInputIsValid(String expected) throws I
     }
 
     @Test
-    void handlerReturnsNonEnrichedBodyWhenEnrichingFails() throws Exception {
+    void handlerIgnoresErrorsWhenTryingToEnrichProjectInformation() throws Exception {
         cristinApiClientStub = spy(cristinApiClientStub);
-        doThrow(new IOException()).when(cristinApiClientStub).getProject(any(), any());
+        doThrow(BadGatewayException.class).when(cristinApiClientStub).getProject(any(), any());
         handler = new FetchCristinProjects(cristinApiClientStub, environment);
+
         GatewayResponse<ProjectsWrapper> response = sendDefaultQuery();
-        var expected = IoUtils.stringFromResources(Path.of(API_RESPONSE_NON_ENRICHED_PROJECTS_JSON));
-        assertEquals(OBJECT_MAPPER.readTree(expected), OBJECT_MAPPER.readTree(response.getBody()));
+
+        assertEquals(HttpURLConnection.HTTP_OK, response.getStatusCode());
+        assertEquals(MediaType.APPLICATION_JSON, response.getHeaders().get(HttpHeaders.CONTENT_TYPE));
     }
 
     @Test
-    public void handlerReturnsOkWhenInputContainsTitleAndLanguage() throws Exception {
+    void handlerReturnsOkWhenInputContainsTitleAndLanguage() throws Exception {
         GatewayResponse<ProjectsWrapper> response = sendDefaultQuery();
         assertEquals(HttpURLConnection.HTTP_OK, response.getStatusCode());
     }
 
     @Test
-    public void handlerIgnoresErrorsWhenTryingToEnrichProjectInformation() throws Exception {
+    void handlerThrowsInternalErrorWhenQueryingProjectsFails() throws Exception {
         cristinApiClientStub = spy(cristinApiClientStub);
-        doThrow(new IOException()).when(cristinApiClientStub).getProject(any(), any());
+        doThrow(new RuntimeException()).when(cristinApiClientStub).queryAndEnrichProjects(any());
         handler = new FetchCristinProjects(cristinApiClientStub, environment);
 
         GatewayResponse<ProjectsWrapper> response = sendDefaultQuery();
```

Comment:
when throwing exceptions we want to be sure that we return Problems. and we should also check the message to the API-client, to be sure that we are not exposing too much information.

Reply:
Should i do something like this: `assertTrue(response.getBody().contains(ERROR_MESSAGE_BACKEND_FETCH_FAILED));`
Or should i write a POJO and deserialize for then to read and compare the detail field? And should that be a separate PR?

Reply:
I thought we were already using zalando Problem. Just leave it as it is and it can be fixed in another PR